src/Traits/AvailableActionTrait.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Opstalent\CrudBundle\Traits;
+
+use Doctrine\Common\Annotations\AnnotationException;
+use Opstalent\CrudBundle\Exception\UnavailableActionException;
+
+/**
+ * Trait AvailableActionTrait
+ * @author Szymon Kunowski <szymon.kunowski@gmail.com>
+ * @package Opstalent\CrudBundle
+ */
+trait AvailableActionTrait
+{
+    /**
+     * @var array
+     */
+    protected $availableActions = ['addable', 'getable', 'listable', 'editable'];
+
+    /**
+     * @param array $availableActions
+     * @return $this
+     */
+    protected function setAvailableActions(array $availableActions)
+    {
+        $this->availableActions = $availableActions;
+        return $this;
+    }
+
+    /**
+     * @param string $action
+     * @return bool
+     * @throws AnnotationException
+     */
+    protected function isActionAvailable(string $action): bool
+    {
+        if (in_array($action, $this->availableActions)) {
+            return true;
+        }
+
+        throw new UnavailableActionException(sprintf(
```

Comment:
this has no sense! it is not consistent!

this method should return `true` or `false`. and do not throw exception here.